api/src/main/java/org/door2door/entities/ContactEntity.java

```diff
@@ -0,0 +1,43 @@
+package org.door2door.entities;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+import java.util.UUID;
+
+@Entity
+@Table(name = "users")
+public class ContactEntity {
```

Comment:
I think we'll need an email and password as well for login purposes

Reply:
We aren't going to do login for the members. The teams will be able to have a password to login to configure this team